File: day14/tests/app-test.py
import unittest
from day14 import app


class Part1(unittest.TestCase):
    def test(self):
        self.assertEqual(8108, app.part1("flqrgnkx"))

class Part2(unittest.TestCase):
    def test(self):
        self.assertEqual(1242, app.part2("flqrgnkx"))

File: day05/src/test/java/AppTest.java
import org.junit.Test;

import java.util.List;

import static org.junit.Assert.*;

public class AppTest {
    private final App app = new App();

    @Test public void testPart1() {
        assertEquals(5, app.part1(List.of(0, 3, 0, 1, -3)));
    }

    @Test public void testPart2() {
        assertEquals(10, app.part2(List.of(0, 3, 0, 1, -3)));
    }
}
